### ISSUE TITLE
explore backward ops, graph training loop, loss function with mix of CPU, TT/GS infra

### ISSUE THREAD

**Issue by muthutt** [tenstorrent/tt-metal#3748]
explore backward ops, graph training loop, loss function with mix of CPU, TT/GS infra
https://github.com/danielsabinasz/TensorSlow
